starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2499,35 +2509,25 @@ sub time2date {
 }
 
 ###################################################################################
-sub count_good_stars{
+sub count_guide_stars{
 ###################################################################################
     my $self=shift;
     my $c;
-    my $clean_acq_count = 0;
-    my $clean_gui_count = 0;
-    $self->{count_nowarn_stars}{ACQ} = $clean_acq_count;
-    $self->{count_nowarn_stars}{GUI} = $clean_gui_count;
+    my $gui_count = 0;
 
     return unless ($c = find_command($self, 'MP_STARCAT'));
     for my $i (1 .. 16){
-        my $type = $c->{"TYPE$i"};
-        next if ($type eq 'NUL');
-        next if ($type eq 'FID');
-	if ($type =~ /ACQ|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_acq_count++;
-	    }
-	}
-	if ($type =~ /GUI|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_gui_count++;
-	    }
+	if ($c->{"TYPE$i"} =~ /GUI|BOT/){
+            my $mag = $c->{"MAG$i"};
+            # Use the fractional magnitudes from ORViewer.  tab-ternary is if/elsif/else
```

Comment:
Tab-ternary is nice!

Reply:
Though I just realized I used the deceptively named "MAG$i" here when we wanted "GS_MAG$i".  Looks like GS_MAG has the right value and MAG has the different value of the 
mean of MINMAG and MAXMAG.  I'm going to rebase fix that.